review/client.go

```diff
@@ -0,0 +1,75 @@
+// Package review provides the /reviews APIs
+package review
+
+import (
+	"net/http"
+
+	stripe "github.com/stripe/stripe-go"
+	"github.com/stripe/stripe-go/form"
+)
+
+// Client is used to invoke /reviews APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// Approve approves a review.
+func Approve(id string, params *stripe.ReviewApproveParams) (*stripe.Review, error) {
+	return getC().Approve(id, params)
+}
+
+// Approve approves a review.
+func (c Client) Approve(id string, params *stripe.ReviewApproveParams) (*stripe.Review, error) {
```

Comment:
Thanks for following convention with the empty params struct here! It feels a little funny to do, but I like that we're staying consistent.

Reply:
yeah and I do feel like we'll have params soon!